src/Symfony/Component/Finder/Finder.php

```diff
@@ -580,14 +581,23 @@ public function sortByModifiedTime(): static
      * The anonymous function receives a \SplFileInfo and must return false
      * to remove files.
      *
+     * @param \Closure(SplFileInfo): bool $closure
+     *
+     * @Symfony7param bool $prune Whether to traverse directories further
```

Comment:
```suggestion
     * @param bool $prune Whether to traverse directories further
```

Reply:
CS Fixer will force it to remove as not declared

Reply:
we accept false positives, like this one is